package.json

```
@@ -22,27 +22,27 @@
     "@dicebear/api-8": "*",
     "@dicebear/api-9": "*",
     "@dicebear/converter": "^9.2.2",
-    "@fastify/cors": "^9.0.1",
+    "@fastify/cors": "^9",
```

This would allow a downgrade. We should avoid that.

```suggestion
    "@fastify/cors": "^9.0.1",
```